### DISCUSSION TITLE
key j/k did not work well in nvterm when to show git log -p or git diff

### DISCUSSION THREAD

**Discussion by cfcy** [NvChad/NvChad#2966] [General]
when I opened the nvterm and tried to show and scroll the results of `git log -p` or `git diff` with key j/k, these two keys did not work well and they were stuck. It only scroll down/up 1 or 2 lines if I pressed j/k serveral times. But at this time the up/down keys worked well. Also in a normal terminal, all worked well.

**Comment by siduck** [comment]
nvterm is depreceated , are you talking about nvchad.term?   Please show steps to reprroduce

just tried :term and k/j dont work on it too :joy: 

just go into normal mode by Ctrl + x and then use k/j

**Reply by cfcy** [reply]
Yes. I think I am using nvchad.term as I am using Nvchad v2.5. I had this problem when I upgraded to Nvchad V2.5. It was OK for old Nvchad(~1 year ago). The reproduce step is quite simple. I have a git repo under a folder. I am using it in WSL.
1. `Alt + i` to open the floating terminal.
2. cd to the git repo and input `git log -p`
3. use the j key to scroll down. 
4. j key will not work. But at this time the Enter key or Down Key work.

**Reply by siduck** [reply]
@cfcy  please go into normal terminal mode by pressing ctrl + x and then k/j keys would work

**Reply by cfcy** [reply]
Hi, Siduck. But I was not able to show more git diff/log using j key. Would you please fix it in the future release? I think it maybe a bug introduced in the new NvChad release. It was OK in old versions.

**Reply by siduck** [reply]
@cfcy  press ctrl + x to go in normal mode. This is not a bug, you cant use     j/k in terminal mode or else how would you type those letters in terminal? ctrl + x mapping will take you back to normal mode, then scroll using j/k like normal buffers

**Reply by cfcy** [reply]
sorry. There may be some misunderstanding. please refer to attachment below. I think j key should be able to show more git log changes.
![2024-08-15-09-08-32-image](https://github.com/user-attachments/assets/68c06176-dec4-4d23-92d1-d56701b1e8e3)

**Reply by siduck** [reply]
https://github.com/user-attachments/assets/16024673-7d9e-4e44-9c4b-6230e1c0bb38

works here @cfcy

**Reply by cfcy** [reply]
Thank you. siduck. Maybe it's related to the system I am using.

**Reply by siduck** [reply]
> Thank you. siduck. Maybe it's related to the system I am using.

try another terminal
